### PR TITLE
Newsweek page

### PR DESCRIPTION
**Pull request by mateomh**
### Newsweek clone page

Created a responsive clone of the **[Newsweek](http://newsweek.com)** landing page using bootstrap for the entire project.

### CODE REVIEW

File: index.html
Comment (expjazz) [inline index.html]: - [x] The Newsweek  logo should be bigger, and the Login and Subscribe button should be vertically aligned with the logo

| Yours | Original |
|:---:|:---:|
|image | image |

#### Medium size screen

- [x] The Newsweek logo should be smaller

- [x] The navbar width should cover the whole viewport

| Yours | Original |
|:---:|:---:|
| image|image |

#### Small screen size

- [x] The right Items have too much space between themselves

- [x]  The navbar should be smaller and the width should cover the whole viewport

| Yours | Original |
|:---:|:---:|
|image|image |

#### Extra small screen size

- [x] All the icons should be smaller, and they should not wrap in a new line.

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html
Comment (expjazz) [inline index.html:58]: - [x] The list containing US, World, Business should be bold, have that lateral border, and have a little more width.

- [x] The "TRENDING" should be black

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html
Comment (expjazz) [inline index.html]: #### In the small screen size

- [x] Pay more attention to the font-sizes, font-weight, and colors to be more close to the original (do not mind the navbar in the original on this screenshot)

| Yours | Original |
|:---:|:---:|
|image|image | image|

#### In the medium screen size

- [x] Same from the above.

| Yours | Original |
|:---:|:---:|
|image|image |

#### General

- [x] All images should contain this little box at the bottom (check in the original to see when it is the case for each image, and the color of the text inside).

image

File: index.html
Comment (expjazz) [inline index.html:527]: - [x] Pay attention to the font sizes, color, and weight.

| Yours | Original |
|:---:|:---:|
|image |image |

File: index.html
Comment (expjazz) [inline index.html]: - [x] You seems to be missing a whole section here

image

File: index.html
Comment (expjazz) [inline index.html]: - [x] It seems you are also missing a lot of content here.
image

File: index.html
Comment (expjazz) [inline index.html]: - [x] The text should wrap around the image, just as is in the original.

| Yours | Original |
|:---:|:---:|
|image|image|

File: index.html
Comment (expjazz) [inline index.html]: #### Small screen size

- [x] The image in the subscribe section is too big. The newsletter section should not have an image.

| Yours | Original |
|:---:|:---:|
|image|image|

#### General

- [x] The resolution of this image seems to be pretty bad because the height is not proportional to the width. Was this in the original? If yes, submit a print screen for the next review or change to the original it is not the case.

image

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### You did a very good job on this project, on all screen sizes. Even though I appointed a few aspects to improve, you should be proud because it really looks good. You also displayed a high level of proficiency in bootstrap. :clap:

However, as I said, there are still a few details to be improved. Most of them you can check in the inline comments directly on the colder below this big box. I will point a few general points here:

- [x] Pay attention to the general font width, sizes, and weight of the fonts, in all screen sizes. This is a small detail that would make your page look even greater.

- [x] The order in the mobile section should be: **Top story => Featured Stories => The Debate => Opinion => Some articles => More Stories => My Turn => Culture and Travel =>  In the magazine.**
In yours is: Top Story => Featured Stories => My Turn => Culture and travel => More Stories => The Debate => Opinion => In the magazine.
#### Try to make as is in the original (You might need to generate extra HTML and use the d-none from bootstrap)

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Comment by krys2fa** [comment]
Hello team,

Nice work with making the requested changes by the previous TSE. Your page looks really good across all screen sizes and you have displayed proficiency in bootstrap in the project.

# Status: Project Approved! 🥇 

Congratulations on getting this project approved. 👍 

I am approving this project, but I suggest you pay attention to font weights and sizes especially with the section headers (i.e capitalized or not).

You can merge this branch (after making the minor changes) into the master branch and go on with your next project.

💡  Do not forget to submit your project completion form after merging this branch.

All the best team,
Chris

**Pull request merged**
